reposilite-site/docs/docker-compose.md

```diff
@@ -0,0 +1,22 @@
+---
+id: docker-compose
+title: Docker Compose
+sidebar_label: Docker Compose
+---
+
+## Installation
+
+See the [docker-compose.yml example file](../docker-compose.yml) for a starting example.
+
+## Notes
+
+The security token (login) is created through a console command. To be able to create a security token with a Docker Compose setup, a little setup is needed:
+
+- add `- REPOSILITE_OPTS=--token admin:secret` to the `docker-compose.yml` file. **NOTE**: this is only needed for the first run and should be removed after creating a real security token!
+- start the container with `docker-compose up -d`
+- access the URL for Reposilite. This depends on the hosts configuration (reverse proxy, etc.)
+- use your credentials from `REPOSILITE_OPTS` to login (ie. admin / secret)
+- open the "Console" tab. **NOTE**: your reverse proxy must also support and accept WebSockets!
+- type `token-generate -s=yourSuperSecretPassword admin m` to create user named "admin" with the password "yourSuperSecretPassword" and **m**anagement permissions
```

Comment:
I think we should avoid `-s` parameter to encourage people to use strong secrets automatically generated through [SecureRandom](https://docs.oracle.com/javase/8/docs/api/java/security/SecureRandom.html) provider (that's default behaviour when you won't specify custom secret. The `-s` parameter is mostly for people that want to migrate secrets from 2.x or they need custom strong secrets for some reason